odp/genetlink.go

```diff
@@ -57,6 +64,9 @@ func (s *NetlinkSocket) LookupGenlFamily(name string) (family GenlFamily, err er
 		return
 	}
 
+	// TODO: It should have been CTRL_CMD_GETFAMILY, though it would be a huge breaking change for
```

Comment:
Have you pinged the kernel maintainers about it?

Reply:
No, do you think we should do that beforehand?

Reply:
No, just suggesting that you could submit a patch to the kernel later on.